src/overlays/gamestates/ovl_opening/z_opening.c

```diff
@@ -1,18 +1,64 @@
 /*
  * File: z_opening.c
  * Overlay: ovl_opening
- * Description:
+ * Description: Initializes the game into the title screen
  */
 
 #include "z_opening.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_opening/Opening_SetupForTitleCutscene.s")
+void Opening_SetupForTitleCutscene(OpeningContext* this) {
+    static s32 openingEntrances[] = { 0x1C00, 0x1C10 };
+    static s32 openingCutscenes[] = { 0xFFFA, 0xFFFA };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_opening/func_80803EA0.s")
+    gSaveContext.eventInf[1] &= 0x7F;
```

Comment:
```suggestion
    gSaveContext.eventInf[1] &= ~0x80;
```
Is this a value or a flag?

Reply:
Pretty sure eventInf is flags, I'll change it.

Reply:
Huh, it needs
```c
    gSaveContext.eventInf[1] &= (u8)~0x80;
```
to match. Is this really better?

Reply:
Yes, it's to be expected. This is quite common, especially for `weekEventReg`. My personal opinion is that while unfortunate, that yes this is better because the `$= ~` is standard for turning off bits, and having it shown explicitly that bit `0x80` is the bit being turned off is clearer imho